stable/elasticsearch/README.md

```diff
@@ -137,6 +137,7 @@ The following table lists the configurable parameters of the elasticsearch chart
 | `data.antiAffinity`                  | Data anti-affinity policy                                           | `soft`                                              |
 | `data.nodeAffinity`                  | Data node affinity policy                                           | `{}`                                                |
 | `data.updateStrategy`                | Data node update strategy policy                                    | `{type: "onDelete"}`                                |
+| `sysctlInitContainer.enabled`        | If true, the sysctl init container is enabledtpl                    | `true`                                              |
```

Comment:
Typo! :)
Also, make sure to state that even if it is false, it does not prevent extraInitContainers to run.